### DISCUSSION TITLE
Please add support for Whisky on macOS

### DISCUSSION THREAD

**Discussion by nonoche2** [Heroic-Games-Launcher/HeroicGamesLauncher#3119] [Ideas]
there is a newcomer on macOS that works with Apple's Game Porting Toolkit, their implementation of DirectX12 on top of Wine. Adding support to it in Heroic on mac would be great

https://github.com/Whisky-App/Whisky

**Comment by Gcenx** [comment]
![IMG_5730](https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/assets/38226388/0a63cf03-c87f-4a47-8b22-9aba64cb15aa)

Along with https://reddit.com/r/macgaming/s/NCesaBlIlj

<br>

All upstream macOS Winehq package as of 8.17 also support D3DMetal you can see information about this https://github.com/Gcenx/macOS_Wine_builds/releases/tag/8.17

From wine-8.18 I’ll be applying additional Rosetta2 specific hacks.
